charts/README.md

```diff
@@ -41,11 +41,62 @@ $ helm uninstall karmada -n karmada-system
 ```
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Example
+### Install agent
+Edited values.yaml
```

Comment:
Don't know how to install agent and estimator. Could you please add some commands?

Reply:
Take a look